config/deploy.rb

```diff
@@ -28,7 +28,7 @@
 namespace :deploy do
   desc 'Restart application'
   task :restart do
-    on roles(:app), in: :sequence, wait: 5 do
```

Comment:
I was trying to make use of the default names available so as to not have to make changes/overrides elsewhere. I'll try using `:app` and `:worker` for role names since that makes more sense.